### DISCUSSION TITLE
Welcome to gQuery Discussions!

### DISCUSSION THREAD

**Discussion by stolinski** [leveluptuts/gQuery#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Brainshack** [comment]
Hey I am just curious what the state of this library is? There are some very small PRs that have been open for over a year? Is this lib dead?

**Comment by tofsjonas** [comment]
Wondering the same. It looks kind of awesome, would be a shame to see it disappear..
